### DISCUSSION TITLE
Engineering goggles release schedule

### DISCUSSION THREAD

**Discussion by quigiboo** [chromiecraft/chromiecraft#4831] [Suggestions]
Currently all engineering goggles are slated to become available alongside tier 5 content, I am already aware that was also the case in classic, where t5 was delayed, and that they were originally released with black temple.

However, I feel that this is a mistake; the engineering trinkets are already overshadowed by the now more easily crafted darkmoon cards, (45 and 51 stamina respectively) and the goggles from this profession will quickly be obsolete to any active raider.

A similar comparison can be drawn to blacksmithing weapons, arguably the flagship item in tbc for blacksmiths, both lionheart blade(ilv107) and its upgrade lionheart champion (ilv123) are craftable without any raid materials and the latter is considered bis until t6. Whereas the engineering goggles(ilv127) have the same heroic dungeon requirement (albeit far fewer of them) but for a much less important slot.

I feel that engineering goggles are an iconic item in tbc, you saw many people using them and for good reason, they're good, they're the reason many people take engineering, I understand not implementing them prior to level 70 but I feel that t5 is too late for them to really shine, t4 would be more appropriate where they would only slightly outperform t4 items at a 125ilv on a minor item slot.

I, like many others took this profession with these goggles in mind, to be used in t4, replaced in t5, then potentially upgraded in sunwell, they’re a major part of the profession for this expansion and without them during appropriate content I would likely drop engineering until wrath.

I kindly and respectfully suggest the release of the base 127ilv goggles be further brought back to t4, or heroic dungeon content. 

Thank you for your time.

**Comment by Cooties-cc** [comment]
There was some oversight and error about the DMF cards actually, in that they too were suppose to be locked till 2.1 and were not in the game from the start. 
![image](https://user-images.githubusercontent.com/62584589/213057977-6d696660-ae96-49a7-bdfb-476fb16099aa.png)
Earliest data entry for them was in june 2007, which may 2007 was patch 2.1. However, I think since it was not caught earlier and people have/are already making decks, they may just have to let the DMF decks slide, since it would be a nightmare to try and remove, refund, and explain to players the change. 

I also understand the comparison between 2nd tier BS weps and how far they carry the player as to the eng goggles doing the same. However I personally don't really like the concept (powerful gear outside a raid that overshadows an entire tier) that you can get an item that only req level 62 + 350 eng, that is as good as most t5 helms. That just seems a bit silly in how powerful these crafts are. Also unlike the BS weps, the goggles would be for *every* class, instead of just a few melee specs. This would ultimately make almost every class that wanted to perform well require eng, similar to shoehorning people into LW for the drums, which I think is bad design. 

As far as I know the goggles were intended as catchup raid gear for new players coming into the scene that couldn't do old raids and wanted to participate in newer raids without much hard filtering. Be that as it may, not everything has to follow the direct footsteps of how things worked in original TBC. 

![image](https://user-images.githubusercontent.com/62584589/213059695-27107db6-8c32-4019-b77a-305e43119746.png)
Comparing the t5 warrior helm to the lvl 62 eng craft, you can see just how close they are, and why it was decided to release them at the more appropriate patch 2.1 schedule. However I think asking for them to released *with* SSC would be your best compromise. This would allow players to stock up the needed materials, and craft the helm before even stepping into SSC to get a t5 helm, without having such a power gap. People can choose to get their t5 head, or have that slot be the one you leave out to retain the 4p bonus. 

To also compare, here is the 2nd tier axe craft to Gorehowl from kara.
![image](https://user-images.githubusercontent.com/62584589/213061195-22bcbe80-4f3f-4cca-805f-1e35ce6a3108.png)
The values are about the same level as comparing the goggles to t5 helm. So while the BS crafts are strong, they are not absurdly stronger than what is avail at that time, unlike the goggles to t4 content. 

Forgot to include a t4 to eng head comparison too, just for more visual data.
![image](https://user-images.githubusercontent.com/62584589/213062774-23210457-bf3d-4e32-a891-462b54930808.png)
Again as mentioned its not by much, but the eng goggles I would use instead of the t5 helm, which means I would wear them from level 62 to t6. That's a bit much for 1 item to me.

**Reply by Jinteki01** [reply]
You keep mentioning the level 62 requirement despite it pretty much meaning nothing in this situation. You need primal nether to craft them, which are BoP and only drop from 70 dungeons or heroics. The argument here is for them to be released either during the heroic phase or the same time as t4 and not during T5 when some, like the tanking one, are irrelevant.

**Reply by quigiboo** [reply]
You are correct that the blacksmithing weapons are only relevant for melee, I would argue most melee, but the axe is probably the weakest of them, with the mace being exceptional for pvp, sword for PVE.

I don't think they were intended as catchup, patch 2.1 in which they were released was a very controversial one, blizzard have gone on record saying BT was prematurely released so I don't think you can tie much of the unrelated content to it.

Tbc is known for its powerful profession items, tailoring has 3 amazing bop sets with 3 items each, for pretty much all casters and many healers as well as the spellstrike set and vengeance wrap tailors can craft themselves 5 items better than anything in t4 with just dungeons.

The darkmoon card situation is understandable, however I think it's unfair to let one slide and not the other. It feels like engineering is being singled out here, I know several people, myself included, have already gathered some materials for the goggles too.
